src/AppBundle/Controller/CommitteeController.php

```diff
@@ -2,36 +2,67 @@
 
 namespace AppBundle\Controller;
 
+use AppBundle\Committee\CommitteePermissions;
 use AppBundle\Entity\Committee;
+use AppBundle\Form\CommitteeFeedMessageType;
 use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method;
 use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
 use Sensio\Bundle\FrameworkExtraBundle\Configuration\Security;
 use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\Form\Extension\Core\Type\SubmitType;
+use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
 
 /**
- * @Route("/comites")
+ * @Route("/comites/{uuid}/{slug}", requirements={"uuid": "%pattern_uuid%"})
  */
 class CommitteeController extends Controller
 {
     /**
-     * @Route("/{uuid}/{slug}", name="app_committee_show", requirements={
-     *   "uuid": "%pattern_uuid%"
-     * })
-     * @Method("GET")
+     * @Route(name="app_committee_show")
+     * @Method("GET|POST")
      * @Security("is_granted('SHOW_COMMITTEE', committee)")
      */
-    public function showAction(Committee $committee): Response
+    public function showAction(Committee $committee, Request $request): Response
     {
+        $form = null;
+        if ($this->isGranted(CommitteePermissions::HOST, $committee)) {
+            $form = $this->createForm(CommitteeFeedMessageType::class)
+                ->add('publish', SubmitType::class, ['label' => 'Publier']);
```

Comment:
Because a submit button is not a controller or model thing. It's a view related thing. If you don't need to check for the button to be clicked, there are absolutely no reasons to add it to the form object.